concepts/examples.py

```diff
@@ -0,0 +1,40 @@
+"""Loading examplary formal contexts."""
+
+import typing
+from urllib.request import urlopen
+
+from .contexts import Context
+
+__all__ = ['load_dataset']
+
+DATASET_SOURCE = "https://raw.githubusercontent.com/fcatools/contexts/main/contexts"
+
+
+# inspired by https://github.com/mwaskom/seaborn/blob/master/seaborn/utils.py#L524
+def load_dataset(name: str, data_src: typing.Optional[str] = DATASET_SOURCE,
```

Comment:
nit: How about keyword-only arguments for eveything except `name` (and maybe that one even positional-only)?

Reply:
Good idea. I implemented this in the latest commits.